app/src/main/java/com/pvp/app/ui/screen/task/TaskScreen.kt

```diff
@@ -1,7 +1,270 @@
+                stringResource(R.string.meal_Preparation),
+                style = TextStyle(
+                    fontWeight = FontWeight.Bold,
+                    fontSize = 17.sp,
+                    color = Color.Black
+                ),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 30.dp)
+            )
+
+            OutlinedTextField(
+                value = preparation,
+                onValueChange = { preparation = it },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 20.dp)
+                    .border(
+                        0.5.dp,
+                        Color.Black,
+                        shape = RoundedCornerShape(5.dp)
+                    ),
+                textStyle = TextStyle(
+                    fontSize = 15.sp
+                )
+            )
+
+            Text(
+                stringResource(R.string.meal_Description),
+                style = TextStyle(
+                    color = Color.Black,
+                    fontWeight = FontWeight.Bold,
+                    fontSize = 17.sp,
+                ),
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 30.dp)
+            )
+
+            OutlinedTextField(
+                value = description,
+                onValueChange = { description = it },
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(top = 20.dp)
+                    .border(
+                        0.5.dp,
+                        Color.Black,
+                        shape = RoundedCornerShape(5.dp)
+                    ),
+                textStyle = TextStyle(
+                    fontSize = 15.sp
+                )
+            )
+
+            Button(
+                onClick = {
+                    val descriptionValue = description.trim()
+                    val durationValue = duration.toLong()
+                    val ingredientsValue = ingredients.trim()
+                    val preparationValue = preparation.trim()
+                    val titleValue = title.trim()
+
+                    if (
+                        titleValue.isNotEmpty() &&
+                        durationValue > 0 &&
+                        (ingredientsValue.isNotEmpty() || preparationValue.isNotEmpty()) &&
+                        descriptionValue.isNotEmpty()
+                    ) {
+                        recipeValue = if (ingredientsValue.isNotEmpty() && preparationValue.isNotEmpty()) {
+                            "$ingredientsValue\n$preparationValue"
+                        } else if (ingredientsValue.isNotEmpty() && preparationValue.isEmpty()) {
+                            ingredientsValue
+                        } else {
+                            preparationValue
+                        }
+                        model.createTaskMeal(
```

Comment:
Different purpose blocks of code should be separated by a single line. One block is assigning value to a variable. Other one is executing some action.